jmix-flowui/flowui/src/main/resources/io/jmix/flowui/view/layout.xsd

```diff
@@ -4221,6 +4221,17 @@
         </xs:complexContent>
     </xs:complexType>
 
+    <xs:complexType name="rangeInputComponent">
+        <xs:complexContent>
+            <xs:extension base="baseHtmlComponent">
```

Comment:
It appears that `RangeInput` doesn't extend `HtmlContainer`, so `baseHtmlComponent` is incorrect base Element.